DESCRIPTION

```diff
@@ -12,11 +15,10 @@ Imports:
     arrow,
     data.table,
     RcppRoll,
-    lubridate,
-    zoo,
-    e1071
```

Comment:
Probably still need this for testing?

Reply:
No, I brought in a copy of skewness package for testing purposes. Otherwise not needed